libraries/rush-lib/src/logic/operations/OperationStatus.ts

```diff
@@ -37,5 +37,9 @@ export enum OperationStatus {
   /**
    * The Operation could not be executed because one or more of its dependencies failed
    */
-  Blocked = 'BLOCKED'
+  Blocked = 'BLOCKED',
+  /**
+   * The Operation had an empty script
+   */
+  EmptyScript = 'EMPTY SCRIPT'
```

Comment:
I'd favor `NoOp` over `EmptyScript` on the basis that operations are no longer required to be a script.